sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/models/WebPubSubResult.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.webpubsub.client.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The result of sending message.
+ */
+@Immutable
+public final class WebPubSubResult {
```

Comment:
Consider adding property `IsDuplicated` as it's a field in `AckMessage` and user may want to know

Reply:
Got it.

This and RejoinGroupFailedEvent seems added in some later version of .NET, and I wasn't noticed.